diff -urN gwaei-1.4.0/configure.ac gwaei-1.4.0.orig/configure.ac
--- gwaei-1.4.0/configure.ac	2010-04-17 20:42:56.938607809 +0200
+++ gwaei-1.4.0.orig/configure.ac	2010-04-17 13:13:57.000000000 +0200
@@ -132,7 +132,8 @@
 AC_CONFIG_MACRO_DIR([m4])
 
 
-AM_CONDITIONAL(MAKE_GNOMEHELP, test x$gnome = xfalse)
+if([test x$gnome = xfalse]);then([cp help/Makefile.am.gnome help/Makefile.am]);fi
+if([test x$gnome = xtrue]);then([cp help/Makefile.am.none help/Makefile.am]);fi
 
 AC_CONFIG_FILES([Makefile src/Makefile mandir/Makefile help/Makefile help/gwaei.omf help/C/gwaei.xml src/gtk-builder-xml/Makefile src/images/Makefile src/schemas/Makefile src/desktop/Makefile src/kpengine/Makefile src/kpengine/conv_jdata.pl rpm/gwaei.spec rpm/rpm_gconf_install_post.sh rpm/rpm_gconf_uninstall_pre.sh rpm/fedora/SPECS/gwaei.spec src/schemas/gwaei.schemas src/include/gwaei/definitions.h po/Makefile.in po/Makefile] Doxyfile)
 AC_OUTPUT
diff -urN gwaei-1.4.0/help/Makefile.am gwaei-1.4.0.orig/help/Makefile.am
--- gwaei-1.4.0/help/Makefile.am	2010-04-17 23:44:39.124476222 +0200
+++ gwaei-1.4.0.orig/help/Makefile.am	2010-04-17 19:44:56.000000000 +0200
@@ -3,8 +3,6 @@
 datadir = @datadir@
 
 
-if MAKE_GNOMEHELP
-
 include $(top_srcdir)/gnome-doc-utils.make
 dist-hook: doc-dist-hook
 DOC_MODULE = gwaei
@@ -22,4 +20,4 @@
 	@ rm -rf Makefile
 	@ rm -rf gwaei.omf
 	@ rm -rf gwaei-C.omf
-endif
+
